### ISSUE TITLE
Missing angular part?

### ISSUE THREAD

**Issue by rain1024** [matt-tyler/oauth2-passport-angular#1]
Can you post angular code in public folder of this project?

I think it's easier for anyone want a try with this demo.

**Comment by imjakechapman** [comment]
Yes please, this is my exact same stack and I can't seem to figure out how to get the page to redirect when I $http.get('my-server-host.com/api/auth/google')
